### ISSUE TITLE
Webhook support

### ISSUE THREAD

**Issue by kevincador** [trakt/trakt-api#10]
Should be configurable for an app with a webhook URL.

Webhook support for things like: new comment, reply, mention, like, friend request,...

Would only call the webhook if the "receiving" user has the app installed.

**Comment by kimdv** [comment]
This could also be done with push notifications. What do you think about that?

**Comment by kevincador** [comment]
This would mean store certificats and pushid of each devices trakt side. I think it's the responsability of the app developer to send the push.

**Comment by lostfocus** [comment]
Plus it's only for mobile apps. Webhooks would be a great solution.

**Comment by janpio** [comment]
How would Trakt know if a users has your app installed or not?

**Comment by kevincador** [comment]
Every app you connect to Trakt (as a user) is listed here: https://trakt.tv/oauth/authorized_applications

As a develeper, you have: https://trakt.tv/oauth/applications

**Comment by janpio** [comment]
Ok, so you would send webhooks for all _connected_ apps, not _installed_ apps.

I fear that would trigger 95% webhook calls for users that are not actually using an app (any more).

**Comment by kevincador** [comment]
Yeah, there's no way to tell if the app is still installed on a mobile device. But webhooks are not only for mobile devices push notifications.

The goal is to be notified of something in real-time instead of pulling the API. Even if you send requests for unintsalled apps, I'm sure the benefit will still be higher.
